File: app/src/main/java/org/scp/gymlog/ui/main/history/TrainingData.java
package org.scp.gymlog.ui.main.history;

import org.scp.gymlog.model.Muscle;

import java.util.Calendar;
import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class TrainingData {
    private List<Muscle> mostUsedMuscles;
    private Calendar startDate;
    private int id;
}

File: app/src/main/java/org/scp/gymlog/ui/main/history/TrainingData.kt
package org.scp.gymlog.ui.main.history

import org.scp.gymlog.model.Muscle
import org.scp.gymlog.model.Training
import java.time.temporal.ChronoUnit

class TrainingData(
    val training: Training,
    val mostUsedMuscles: List<Muscle>
) {
    val duration: Int?
        get() = training.end?.let { ChronoUnit.MINUTES.between(training.start, it).toInt() }
}
